source/platform/native.rst

```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, the under-development Velocity 1.1.0 and Sponge 8 support Adventure
+As of this writing, the under-development Velocity 1.1.0, Sponge 8 and Paper 1.16.5 support Adventure
```

Comment:
```suggestion
As of this writing, the Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
```

Reply:
maybe its my dum english, but that `the` in there sounds dum if you remove the under development

Reply:
Sounds weird for me too. Either remove the `the` or add platforms/server softwares in the end. So like `As of this writing, the Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ platforms` or `As of this writing, the Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ server softwares`.